packages/react-swipeable-views/src/SwipeableViews.js

```diff
@@ -389,6 +389,14 @@ class SwipeableViews extends Component {
         passive: false,
       },
     );
+    this.mouseMoveListener = addEventListenerEnhanced(this.rootNode, 'mousemove', (e) => {
```

Comment:
Do we need the event to be `active` (passive: false)? I would expect a regular `onMouseMove` callback property to be enough.